pagopa/io-dev-api-server
# chore: [IOBP-698] Add new mocked payment method of method management type REDIRECT

## Short description
This PR adds a new mock in the payment methods available and not onboardable with the method management type REDIRECT

## List of changes proposed in this pull request
- Added a new item to the list of available payment methods

## How to test
- Start it with the payment flow in the IO app, you should be able to see a new payment method in the list (i.e. PostePay)
